app/assets/javascript/application.js

```diff
@@ -9,6 +10,12 @@ jQuery(function($) {
   });
 
   ////
+  // Add "select all"/"clear all" buttons to each select2 dropdown menu
+  GOVUK.DropdownSelectAll({
+    $el: $("select.select2")
```

Comment:
Is it safe to apply this to every select2 in the app? Could add another class for this instead.